test/vmtester/vmtester.cpp

```diff
@@ -65,7 +65,8 @@ class cli_parser
                 arguments.emplace_back(std::move(arg));
                 continue;
             }
-            else if (x <= 2)
+
+            if (x <= 2)
```

Comment:
The `else if` version is more readable, though the `continue` statement above makes sure the new version is semantically equivalent.

Reply:
This is this check: https://clang.llvm.org/extra/clang-tidy/checks/readability-else-after-return.html.